package.json

```diff
@@ -10,7 +10,7 @@
     "ember-addon"
   ],
   "ember-addon": {
-    "before": "serve-files-middleware"
+    "before": "ember-cli:broccoli:watcher"
```

Comment:
Hmm, I _believe_ that we can just list them both in an array.  Only one will have an effect, but it shouldn't error...

```
"before": ["serve-files-middleware", "ember-cli:broccoli:watcher"]
```